git_flow.rst

```diff
@@ -11,71 +11,106 @@ code. Developers_ want to make some changes to the code. Maintainers_ are the
 ones who make sure the changes to the code make sense, and incorporate them
 into the code base. We will talk about each of these use cases in turn.
 
-One of the most challenging concepts with :bash:`git` source code control
-is the fact that its job is to seamlessly manage multiple different versions
-of a code base. The idea is that :bash:`git` allows you to swap back and
-forth between these different versions while working within the same directory
-structure.
-
-In other words, typically there is only one repository for a given project on
-a user's local machine. That repository can have multiple branches within it.
-When a user switches between branches, they are effectively switching between
-different sets of files within that repository. The :bash:`git` protocol thus
-modifies or updates files in-place for users.
-
-In addition to each user having a local git repository, they usually become
-associated with some number of remote repositories on GitHub. The user's local
-repository can "know" about any number of remote repositories, and users get 
-to call those remote repositories whatever they want. Here is a diagram of the
-connections between the remote repositories on GitHub and the local computers 
-of each of three different users:
-
-.. _diagram:
-.. image:: https://cdn.rawgit.com/LABSN/expyfun/master/doc/git_flow.svg
-
-
 Users
 ^^^^^
-Users who want access to the source code typically do something like this::
+If you want to use software that is hosted on GitHub and get updates when they
+are released, but you don't think it's likely that you will ever want to fix
+bugs or add new features yourself, then probably you are a User (you can upgrade
+yourself to a Developer later if you need to).
+
+Users will want to take the "official" version of the software, make a copy of
+it on their own computer, and run the code from there. Using ``expyfun`` as an
+example, this is done on the command line like this::
 
     $ git clone git://github.com/LABSN/expyfun.git
     $ cd expyfun
     $ python setup.py install
 
-Perhaps without realizing it, these users have set up a structure so that
-their local repository knows about a remote repository named "origin" that
-points to `<http://github.com/LABSN/expyfun.git>`_, and they have checked out
-the branch "master" from that repository. This can be seen by doing::
+This will create a folder ``expyfun`` in the current directory to store the
+source code, and install the necessary parts in the special places where python
+looks when you do :python:`import expyfun` and call ``expyfun`` functions in
```

Comment:
if you want to do an inline `:python:` (which makes sense), you need to add a directive like the one I made for bash at the top of the file